src/viam/ur/module/ur_arm.cpp

```diff
@@ -573,8 +591,16 @@ void URArm::move(std::vector<Eigen::VectorXd> waypoints, std::chrono::millisecon
         if (std::isinf(duration)) {
             throw std::runtime_error("trajectory.getDuration() was infinite");
         }
+        if (std::isnan(duration)) {
+            throw std::runtime_error("trajectory.getDuration() was nan");
+        }
+        if (duration > 600) {  // if the duration is longer than 10 minutes
```

Comment:
This feels arbitrary. Should it be a configuration parameter?

Reply:
def arbitrary, came up with this number when talking to rand. I can go ahead and make it configurable

Reply:
Let's ticket it, TODO it, and leave this as is for purposes of this review.